### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by DeekshithSN**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM ubuntu
```
Comment (Priyakumaraguru) [inline Dockerfile]: please chanage to centos

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+RUN mkidr test
```
Comment (litu54) [inline Dockerfile]: Please add some other directory

**Review by Priyakumaraguru — APPROVED** [review]
LGTM

**Review by arpita-hajare — APPROVED** [review]
LGTM

**Pull request merged**